src/helpers/network/getContractName.ts

```diff
@@ -0,0 +1,18 @@
+import { useSnapshot } from 'valtio'
+import ContractMetadataStore from 'stores/ContractMetadataStore'
+import Network from 'models/Network'
+import networks from 'networks'
+import prettifyContractName from 'helpers/network/prettifyContractName'
+
+export default function (address: string, network: Network, truncate = false) {
```

Comment:
Maybe let's make an object argument? Because you need to check the order each time you want do use the helper